<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPersoDoor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Beenden</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelbso">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>buergerserviceorg.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="adduButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Neuer Benutzer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deluButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Benutzer Löschen</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>591</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Starten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stoppen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Benutzeradministration:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>persodoor.log (last 7 lines):</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>591</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionHilfe"/>
    <addaction name="actionUeber"/>
    <addaction name="actionZertifikat"/>
   </widget>
   <widget class="QMenu" name="menuPIN">
    <property name="title">
     <string>PIN</string>
    </property>
    <addaction name="actionEdit_PIN"/>
   </widget>
   <addaction name="menuPIN"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionUeber">
   <property name="text">
    <string>Ueber</string>
   </property>
  </action>
  <action name="actionZertifikat">
   <property name="text">
    <string>Zertifikat</string>
   </property>
  </action>
  <action name="actionEdit_PIN">
   <property name="text">
    <string>Ändern default PIN</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
